robottelo/hosts.py

```diff
@@ -1756,6 +1756,7 @@ def __init__(self, hostname=None, **kwargs):
         # create dummy classes for later population
         self._api = type('api', (), {'_configured': False})
         self._cli = type('cli', (), {'_configured': False})
+        self._ui_session = None
```

Comment:
non-blocking: This attribute is not used anywhere.

Reply:
Haa! Keep forgetting things since I am switching the context here a lot !